specs/pages/bracket.md

```diff
@@ -0,0 +1,9 @@
+# tar
+
+> Archiving utility.
+> Often combined with a compression method, such as gzip or bzip.
+> More information: <https://www.gnu.org/software/tar>.
+
+- [C]reate an archive from [f]iles:
```

Comment:
```suggestion
- [c]reate an archive from [f]iles:
```

Reply:
If `c` is the first letter in the sentence, why should be in lower case?

Reply:
To follow the name of the option (`c`), since some commands can also have uppercase options. But mostly because you used it like this in 015.md, so I just wanted it to be consistent.